package.json

```
@@ -43,7 +43,7 @@
     "react-router-dom": "^4.3.1",
     "react-select": "^3.1.0",
     "react-table": "^6.10.0",
-    "react-variant-view": "0.1.3-beta",
+    "react-variant-view": "0.1.10",
```

Probably upgrading this to the latest versions will fix the build issues:

```suggestion
    "react-variant-view": "^0.1.11",
```

We should also upgrade `genome-nexus-ts-api-client` to `^1.1.12`